### PR TITLE
S3 prep

### PR DESCRIPTION
**Pull request by KimarieHB**
Add build script for production

### CODE REVIEW

File: data/sampleData.json [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+          {
+            "track_number" : "20",
+            "name" : "et dolor doloremque",
+            "song_url" : "https://viola.org"
+          }
+        ]
+      }
+    ]
+  }
+]
```
Comment (theofficetroll) [inline data/sampleData.json:427]: Looks good.

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "react-dev": "webpack -d --watch",
+    "build": "webpack",
```
Comment (theofficetroll) [inline package.json]: You might want to consider using production mode here, but this should be fine for now.

https://webpack.js.org/configuration/mode/

File: webpack.config.js [hunk 3]
```diff
@@ -20,7 +20,11 @@ module.exports = {
         options: {
           presets: ['@babel/preset-react', ['@babel/preset-env', {'targets': 'defaults'}]],
         },
-      }
+      },
+      {
+        test: /.json$/,
+        loader: 'json-loader',
+      },
```
Comment (theofficetroll) [inline webpack.config.js:27]: Makes sense.

**Review by theofficetroll — APPROVED** [review]
Looks fine.
Left a comment about your build script and a link that may help.

File: package.json (hunk 2 above)
Comment (KimarieHB) [reply package.json]: Thanks! That is very good! Will do!

**Pull request merged**